SU-SWS/saa_victoria_subtheme
# ADAPT-703: Added map link to use cta mixin.

# ALMOST READY FOR REVIEW

# Summary
- Set map link to use the cta mixin so the vivid red is the RIGHT vivid red.

# Review By (Date)
- 9/17

# Urgency
- medium, before launch

# Steps to Test

1. Pull down the branch or test on https://saauserguide-test.sites.stanford.edu/
2. Clear the site cache
3. Test that the map link in the first column of the footer uses the correct red to match the other links below it: it should be using #CE0D0D
4. Test that the red links below the map look like the correct red (#CE0D0D) and then cardinal red on hover.

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket ADAPT-703
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
